pkg/cacheutil/memcached_client.go

```diff
@@ -297,6 +300,13 @@ func (c *memcachedClient) SetAsync(_ context.Context, key string, value []byte,
 
 		c.duration.WithLabelValues(opSet).Observe(time.Since(start).Seconds())
 	})
+
+	if err == errMemcachedAsyncBufferFull {
+		c.skipped.WithLabelValues(opSet, reasonAsyncBufferFull).Inc()
+		level.Debug(c.logger).Log("msg", "failed to store item to memcached, check your configuration", "err", err, "size", len(c.asyncQueue))
```

Comment:
```suggestion
		level.Debug(c.logger).Log("msg", "failed to store item to memcached because the async buffer is full", "err", err, "size", len(c.asyncQueue))
```